docs/product-installation-guide/src/main/asciidoc/assembly_installing-on-eap.adoc

```diff
@@ -10,7 +10,7 @@ For information on how to install {EAP} 7.0.7 on your computer, see the https://
 include::eap-business-central-install-proc.adoc[leveloffset=+1]
```

Comment:
Just a general comment first that I'm not sure why in the prereqs there are ifdef conditions for the required .zip file downloads depending on BPM Suite vs BRMS, when other times there is no ifdef for the same mentioned files in the instructions. No to mention that all the sections/chapters are explicitly for BPM Suite. So not sure why we'd ifdef for BRMS anyway. In short, seems to me that we either need to ifdef everything that depends on product (not just some things) or ifdef nothing and make this just for only BPM Suite. I likely have misunderstood, but that's what I gather.

Reply:
@sterobin  When Sheldon started this doc, he added the BPMS/BRMS if defs. For LA, we are only producing and only have actual file names for BPMS. But rather than strip out the existing ifdefs I decided to keep them and update them later, when we have actual file names for BRMS.